### PR TITLE
Add more scripts for testing manipulations with csm and node operators

### PR DESCRIPTION
**Pull request by skhomuti**

### CODE REVIEW

File: script/fork-helpers/NodeOperators.s.sol [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    ForkHelpersCommon,
+    Utilities
+{
+    modifier broadcastPenaltyReporter() {
+        _setUp();
+        address penaltyReporter = csm.getRoleMember(
+            csm.REPORT_EL_REWARDS_STEALING_PENALTY_ROLE(),
+            0
+        );
+        _setBalance(penaltyReporter, "1000000000000000000");
```
Comment (vgorkavenko) [inline script/fork-helpers/NodeOperators.s.sol:24]: can the huge value be default in `_setBalance`?

File: script/fork-helpers/NodeOperators.s.sol [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+        uint256 limit
+    ) external broadcastStakingRouter {
+        csm.updateTargetValidatorsLimits(noId, targetLimitMode, limit);
+    }
+
+    function reportStealing(
+        uint256 noId,
+        uint256 amount
+    ) external broadcastPenaltyReporter {
+        csm.reportELRewardsStealingPenalty(noId, "", amount);
```
Comment (vgorkavenko) [inline script/fork-helpers/NodeOperators.s.sol:121]: hash might be useful for notification testing, i guess. let's set some mocked value

File: script/fork-helpers/NodeOperators.s.sol [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    }
+
+    function cancelStealing(
+        uint256 noId,
+        uint256 amount
+    ) external broadcastPenaltyReporter {
+        csm.cancelELRewardsStealingPenalty(noId, amount);
+    }
+
+    function settleStealing(uint256 noId) external broadcastPenaltySettler {
```
Comment (vgorkavenko) [inline script/fork-helpers/NodeOperators.s.sol:131]: do we need `compensate` script as well?

File: script/fork-helpers/NodeOperators.s.sol [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    function exitRequest(
+        uint256 noId,
+        uint256 validatorIndex,
+        bytes calldata validatorPubKey
+    ) external {
+        _setUp();
+        IVEBO vebo = IVEBO(locator.validatorsExitBusOracle());
+        bytes memory data;
+
+        bytes3 moduleId = bytes3(uint24(4));
```
Comment (vgorkavenko) [inline script/fork-helpers/NodeOperators.s.sol:146]: dangerous magic number

File: script/fork-helpers/PauseResume.s.sol [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        vm.startBroadcast(accountingAdmin);
+        _;
+        vm.stopBroadcast();
+    }
+
+    function pauseCSM() external broadcastCSMAdmin {
+        csm.grantRole(csm.PAUSE_ROLE(), csmAdmin);
+        csm.pauseFor(type(uint256).max);
+
+        assertTrue(csm.isPaused());
```
Comment (vgorkavenko) [inline script/fork-helpers/PauseResume.s.sol:34]: if we have asserts here, do we need the same in another script (`NodeOperators.s.sol`)?

File: script/fork-helpers/NodeOperators.s.sol (hunk 4 above)
Comment (skhomuti) [reply script/fork-helpers/NodeOperators.s.sol:146]: forget to fix it, thx

File: script/fork-helpers/PauseResume.s.sol (hunk 5 above)
Comment (skhomuti) [reply script/fork-helpers/PauseResume.s.sol:34]: yep!

**Review by exromany — DISMISSED** [review]
looks good for me

**Review by vgorkavenko — DISMISSED** [review]

**Review by vgorkavenko — APPROVED** [review]

**Review by dgusakov — DISMISSED** [review]

**Review by dgusakov — APPROVED** [review]

**Review by exromany — APPROVED** [review]

**Review by dgusakov — APPROVED** [review]

**Pull request merged**